mon-pix/tests/unit/routes/account-recovery/update-sco-record-test.js

```diff
@@ -150,11 +150,85 @@ describe('Unit | Route | account-recovery | update sco record', function() {
           // then
           return promise.then((result) => {
             expect(result.errorMessage).to.equal(this.intl.t('api-error-messages.internal-server-error'));
-            expect(result.showReturnToHomeButton).to.be.true;
+            expect(result.showBackToHomeButton).to.be.true;
           });
         });
       });
+    });
+  });
+
+  describe('#authenticateUser', function() {
+
+    it('should call session authenticate method with password and email', async function() {
+      // given
+      const email = 'user@example.net';
+      const password = 'Password123';
+      const scope = 'mon-pix';
+
+      const sessionStub = Service.create({
+        authenticate: sinon.stub(),
+      });
+
+      const route = this.owner.lookup('route:account-recovery/update-sco-record');
+      route.session = sessionStub;
+
+      // when
+      await route.authenticateUser({ email, password });
+
+      // then
+      sinon.assert.calledWith(route.session.authenticate, 'authenticator:oauth2', {
+        login: email,
+        password,
+        scope,
+      });
+    });
+
+    context('when user is already authenticated', function() {
 
+      it('should load currentUser and replace by index route', async function() {
```

Comment:
L'assertion du test correspond à une vérification de la déconnexion de l'utilisateur ?
      it('should call invalidate user session', async function() {

Reply:
Done